src/credentials/credentialsUtilities.ts

```diff
@@ -0,0 +1,19 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Credentials } from 'aws-sdk'
+
+export function asEnvironmentVariables(credentials: Credentials): NodeJS.ProcessEnv {
+    const environmentVariables: NodeJS.ProcessEnv = {}
+
+    environmentVariables.AWS_ACCESS_KEY = credentials.accessKeyId
+    environmentVariables.AWS_ACCESS_KEY_ID = credentials.accessKeyId
+    environmentVariables.AWS_SECRET_KEY = credentials.secretAccessKey
+    environmentVariables.AWS_SECRET_ACCESS_KEY = credentials.secretAccessKey
+    environmentVariables.AWS_SESSION_TOKEN = credentials.sessionToken
+    environmentVariables.AWS_SECURITY_TOKEN = credentials.sessionToken
```

Comment:
Why the duplication?

Reply:
See https://github.com/aws/aws-toolkit-jetbrains/blob/master/jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/CredentialManager.kt#L81

Reply:
Some of the legacy options: https://github.com/boto/botocore/blob/c76553d3158b083d818f88c898d8f6d7918478fd/botocore/credentials.py#L256